packages/richtext-lexical/src/features/toolbars/shared/ToolbarDropdown/index.scss

```diff
@@ -88,6 +88,10 @@
       &.active {
         background-color: var(--theme-elevation-100);
       }
+      &.disabled {
+        cursor: not-allowed;
+        opacity: 0.2;
+      }
```

Comment:
I have copied the styles from popup__dropdown, or from the single button elements of the toolbar when they are disabled. See for example line 24 of this same file.